marshaller.go

```diff
@@ -162,18 +200,49 @@ func buildEqualsClause(v interface{}, fieldName string) string {
 	return constructEqualsClause(v, fieldName, equalsOperator)
 }
 
+func buildGreaterThanClause(v interface{}, fieldName string) string {
+	return constructEqualsClause(v, fieldName, greaterThanOperator)
```

Comment:
Can we rename `constructEqualsClause` to `constructConditionalClause` or something that depicts that it is constructing more than just equals clause?

Reply:
How does `constructComparisonClause` sound?